### ISSUE TITLE
Performance Question.

### ISSUE THREAD

**Issue by Minseojeonn** [Kaike-Zhang/PLD#3]
Hello, and thank you for your contributions to this paper.

I have a question regarding the performance results reported.
Using the provided code and tuning within the hyperparameter ranges suggested in the paper,
I conducted experiments across five different random seeds (2024 to 2028).

In particular, when using LightGCN with 3 layers, a learning rate of 0.01, weight decay of 0.0001, and seed 2024 on the MIND dataset,
I observed that your proposed method achieved higher performance than what was reported in the paper.
(Please see the attached images for details.)
Given the hyperparameter space outlined in your paper, the reported performance of your method might be slightly underestimated.

Would you be able to kindly clarify any possible reasons for this discrepancy?
I would greatly appreciate any insights you could provide.

Thank you very much for your time and consideration.

<img width="537" alt="Image" src="https://github.com/user-attachments/assets/6fd2b30e-9748-48de-82c6-a69ed52e8294" />

<img width="313" alt="Image" src="https://github.com/user-attachments/assets/77821ae5-e874-417f-95c0-6d23504d96a8" />

- LightGCN, 3 Layers, lr = 0.01, decay = 0.0001, seed 2024, MIND

**Comment by Minseojeonn** [comment]
Based on additional experiments I conducted, I found that tuning the suggested method with a learning rate between 0.001 and 0.00001, and a decay rate between 0.001 and 0.00001, yielded similar results.
Could you kindly confirm if the tuning space mentioned in the paper is correct?

**Comment by Kaike-Zhang** [comment]
Thank you for your interest and for carefully reproducing the results!

Regarding your observation, the discrepancy might be due to the following reasons:

1. **Search Strategy**: To ensure fair comparison and reduce the overall search space, we first tuned the base hyperparameters (e.g., learning rate and weight decay) on the backbone model (e.g., LightGCN) alone. Then, we used these fixed backbone hyperparameters when evaluating our method (PLD), tuning only the additional PLD-specific parameters such as the candidate pool size and the temperature coefficient $\tau$. It’s possible that jointly tuning all hyperparameters together could yield even better performance for PLD.

2. **System Variability**: Differences in GPU models, PyTorch versions, and other system-level settings can also lead to slight variations in performance, especially for methods sensitive to numerical stability.

We appreciate you bringing this to our attention and providing detailed experimental results. Let us know if you have any further questions or findings!

**Comment by Minseojeonn** [comment]
I sincerely appreciate your kind and insightful response!
